run-test-coverage.sh

```diff
@@ -0,0 +1,23 @@
+cd users
+pytest 					\
+	--cov-report term-missing 	\
+	--cov-config ../coverage-config \
+	--cov=.
+rm .coverage
+cd ..
+
+cd employees
+pytest 					\
+	--cov-report term-missing 	\
+	--cov-config ../coverage-config \
+	--cov=.
+rm .coverage
+cd ..
+
+cd managers
+pytest 					\
+	--cov-report term-missing 	\
+	--cov-config ../coverage-config \
+	--cov=.
+rm .coverage
+cd ..
```

Comment:
All of these still does not exist. Delete them. Add comment that it needs to be updated when any of these modules will be added.

Reply:
Done